### PR TITLE
SO changed bday

### PR DESCRIPTION
**Pull request by SaliasO**

### CODE REVIEW

File: data/birth-dates/09.txt [hunk 1]
```diff
@@ -1 +1 @@
-Noam Chomsky: 7 December ????
+Noam Chomsky: 7 December 7. Dezember 1928
```
Comment (zuphilip) [inline data/birth-dates/09.txt:1]:
```suggestion
Noam Chomsky: 7 December 1928
```

**Comment by zuphilip** [comment]
You doubled the data here, but we only need to add the year. See my suggested change. @SaliasO Do you want to fix this in another commit and push it again? We can reuse the same pull request. Just update your fork and write something here for pinging us to look again.

**Comment by SaliasO** [comment]
done :-)

**Review by stweil — APPROVED** [review]
Now it looks good, thank you.

**Comment by stweil** [comment]
For real world pull requests the subject line could be still improved.

**Pull request merged**